pinot-server/src/main/java/com/linkedin/pinot/server/starter/helix/HelixServerStarter.java

```diff
@@ -268,44 +245,10 @@ private void setupHelixSystemProperties(Configuration conf) {
   public void stop() {
     _adminApiApplication.stop();
     setShuttingDownStatus(true);
-    if (_pinotHelixProperties.getBoolean(CommonConstants.Server.CONFIG_OF_ENABLE_SHUTDOWN_DELAY, true)) {
+    if (_helixServerConfig.getBoolean(CommonConstants.Server.CONFIG_OF_ENABLE_SHUTDOWN_DELAY, true)) {
       Uninterruptibles.sleepUninterruptibly(_maxQueryTimeMs, TimeUnit.MILLISECONDS);
     }
     _helixManager.disconnect();
     _serverInstance.shutDown();
   }
-
-  public static HelixServerStarter startDefault() throws Exception {
```

Comment:
This is useful code to keep around, while needing to debug in production.

Reply:
I don't quite understand why this can be used in production for debugging. We start a local zookeeper and set up a cluster called "quickstart" in prod?